src/templates/DatasetSearch/index.jsx

```diff
@@ -83,15 +83,21 @@ const DatasetSearch = ({
   },[fulltext, selectedFacets, sort])
   useEffect(() => {
     const baseNumber = Number(totalItems) > 0 ? 1 : 0;
-    const startingNumber = baseNumber + ((Number(pageSize) * Number(page + 1)) - Number(pageSize))
-    const endingNumber = (Number(pageSize) * Number(page + 1)); 
+    const startingNumber = baseNumber + ((Number(pageSize) * Number(page)) - Number(pageSize))
+    const endingNumber = (Number(pageSize) * Number(page)); 
     setCurrentResultNumbers({
       total: Number(totalItems),
       startingNumber: startingNumber,
-      endingNumber: endingNumber
+      //????
```

Comment:
forgot this :)

Reply:
Also this ending number thing exists on dataset preview too. Not sure if you want to just fix that in this PR too.

Reply:
It looks like dataset preview is pulling from data-catalog-components, so that would need to be a new fix there. I'll leave it for now. Maybe make a spinoff ticket in Jira